core/fishing_direct_reply.py

```
@@ -50,6 +50,35 @@ async def generate_reply_strategy(self, chat_id: str, event, trigger_type: str)
+                )
+                if hasattr(event, "set_extra"):
+                    event.set_extra(
+                        "angelheart_decision_context",
+                        {
+                            "historical_context": historical_context,
+                            "recent_dialogue": recent_dialogue,
+                            "boundary_ts": boundary_ts,
+                            "boundary_message_id": boundary_message_id,
+                        },
+                    )
+            except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)